pkg/network/common/validation.go

```diff
@@ -140,8 +140,8 @@ func ValidateHostSubnet(hs *osdnv1.HostSubnet) error {
 			allErrs = append(allErrs, field.Invalid(field.NewPath("subnet"), hs.Subnet, err.Error()))
 		}
 	}
-	// In theory this has to be IPv4, but it's possible some clusters might be limping along with IPv6 values?
-	if net.ParseIP(hs.HostIP) == nil {
+	_, err := validateIPv4(hs.HostIP)
```

Comment:
> But in OCP 4.x, the HostSubnet CRD requires the field to be IPv4.

Nit: do we need to validate it then? There shouldn't really be any case where that is not already upheld by the CRD validator (well, unless someone removes it or writes an incorrect regular expression when trying to match IPv4....yeah, maybe let's keep it)